build/tasks/release-checks/tests/generic/au-cypress.js

```diff
@@ -30,7 +30,7 @@ class AuCypressRunsTests extends Test {
   }
 
   execute() {
-    this.executeCommand = new ExecuteCommand('au', ['run'], (msg) => this.onOutput(msg));
+    this.executeCommand = new ExecuteCommand('npm', ['start'], (msg) => this.onOutput(msg));
```

Comment:
Since you normalised `au` commands to behind same npm scripts, can you update all the file names from `au-xxx.js` to maybe `xxx.js` because `au-` is misleading for the new webpack setup.

Reply:
Absolutely. Thanks for pointing out. Will correct this.

Reply:
Correction done